src/Microsoft.AspNetCore.BuildTools.CreateLZMA/sdk/Sdk.Targets

```diff
@@ -0,0 +1,2 @@
+<Project>
```

Comment:
Without this empty file, the SDK can't be used by the test project.

Reply:
Add this comment to source code